.CI/build_all

```diff
@@ -19,7 +19,7 @@ conda install --yes --quiet conda-forge-build-setup
 source run_conda_forge_build_setup
 
 # install conda-build 2.x to build with a long prefix
-conda install --yes --quiet conda-build=2
+conda install --yes --quiet conda-build=2.0.12
```

Comment:
I don't think this repo is using conda-build=2, even though that line is there. Previously it was downgraded to 1.21.
```
The following packages will be DOWNGRADED due to dependency conflicts:

    conda-build: 2.1.0-py35_0 defaults --> 1.21.14-py35_0 defaults
```
When pinned to 2.0.12 instead of 2, it actually works.
cc @ocefpaf